src/app/app.tsx

```diff
@@ -27,8 +25,8 @@ export function App() {
       <PersistGate loading={<FullPageLoadingSpinner />} persistor={persistor}>
         <HeadProvider />
         {/* TODO: this works but investigate importing radixBaseCSS in panda layer config */}
-        <ThemeProvider theme={theme} css={radixBaseCSS}>
-          <ThemeSwitcherProvider>
+        <ThemeSwitcherProvider>
+          <styled.div css={radixBaseCSS}>
```

Comment:
I still need to load this `radixBaseCSS` here. I think we should be able to load this in `panda` but I couldn't figure out how